google/ggrc-core
# GGRC-5459 Tabs in HNB are displayed with latency on the Audit page

# Issue description

Low performance for loading HNB in Audit and ~~Assessment~~.

**Need to add changes for frontend part.**

# Steps to test the changes

Steps to reproduce:
1. Open any Audit page with Assessment and look at the HNB: tabs are displayed with latency.
2. ~~Open Assessment in a separate tab: tabs are displayed with latency.~~

Actual Result: Tabs in HNB are displayed with latency on the Audit page.

Expected Result: Improve performance of displaying Tabs in HNB on the Audit page.

# Solution description

Add `snapshot_counts` endpoint for Audit.
Add test for check this functionality.

# Sanity checklist




# PR Review checklist
